backend/code_review_backend/issues/serializers_v2.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from code_review_backend.issues.models import IssueLink
+
+
+class IssueLinkHashSerializer(serializers.ModelSerializer):
+    """Serialize an Issue hash from the IssueLink M2M"""
+
+    id = serializers.UUIDField(source="issue_id")
```

Comment:
Note: add `read_only` on each field as it will not be applied from the Meta class.